.tekton/update-dependencies.yaml

```diff
@@ -21,219 +24,8 @@ spec:
-        workspaces:
-          - name: source
-            workspace: source
-        params:
-          - name: repo_name
-            value: $(params.repo_name)
-          - name: repo_owner
-            value: $(params.repo_owner)
-          - name: source_branch
-            value: $(params.source_branch)
-          - name: target_branch
-            value: "robot/$(params.source_branch)/update_binaries"
-        taskSpec:
-          params:
-            - name: repo_name
-            - name: repo_owner
-            - name: source_branch
-            - name: target_branch
-          results:
-            - name: skip-open-pr
-              description: Flag that controls whether or not to open a PR
-              type: string
-          steps:
-            - name: update-binaries
-              image: quay.io/redhat-pipeline-service/dependencies-update:$(params.source_branch)
-              imagePullPolicy: Always
-              workingDir: $(workspaces.source.path)
-              command:
-                - ./developer/images/dependencies-update/hack/bin/update.sh
-                - --commit_to
-                - "$(params.target_branch)"
-                - --task
-                - update_binaries
-                - --workspace_dir
-                - $(workspaces.source.path)
-            - name: is-pr-skipped
-              image: quay.io/redhat-pipeline-service/dependencies-update:$(params.source_branch)
-              imagePullPolicy: Always
-              env:
-                - name: GITHUB_TOKEN
-                  valueFrom:
-                    secretKeyRef:
-                      name: github
-                      key: token
-              workingDir: $(workspaces.source.path)
-              command:
-                - ./developer/images/dependencies-update/hack/bin/is-pr-skipped.sh
-                - --debug
-                - --name
-                - $(params.repo_name)
-                - --owner
-                - $(params.repo_owner)
-                - --target_branch
-                - $(params.target_branch)
-                - --result
-                - $(results.skip-open-pr.path)
-          workspaces:
-            - name: source
-      - name: open-pr-binaries
-        when:
-          - input: "$(tasks.update-binaries.results.skip-open-pr)"
-            operator: in
-            values: ["no"]
-        runAfter:
-          - update-binaries
-        taskRef:
-          name: github-open-pr
-          kind: Task
-        params:
-          - name: REPO_FULL_NAME
-            value: "$(params.repo_owner)/$(params.repo_name)"
-          - name: BASE
-            value: $(params.source_branch)
-          - name: HEAD
-            value: robot/$(params.source_branch)/update_binaries
-          - name: BODY
-            value: Generated by developer/images/dependencies-update/hack/bin/update.sh
-          - name: TITLE
-            value: Update binaries
   workspaces:
-    - name: source
+    - name: workdir
```

Comment:
same as above, it doesn't need to create workspace

Reply:
See previous answer about the task being run multiple times in a single pipeline.